dencoding/yaml_decoder_test.go

```diff
@@ -98,12 +98,16 @@ key2: value6
 	})
 
 	t.Run("YamlAliases", func(t *testing.T) {
-		b := []byte(`foo: &foo
+		b := []byte(`foo: &foofoo
   bar: 1
-  baz: "baz"
+  baz: &baz "baz"
 spam:
   ham: "eggs"
-  <<: *foo
+  bar: 0
+  <<: *foofoo
+  baz: "bazbaz"
```

Comment:
This makes sure that the `*foofoo` dereferencing

1. overrides the `bar: 0` value
2. does not override the `baz: "bazbaz"` value